parsers/jscript.c

```diff
@@ -1653,15 +1653,45 @@ static bool parseMethods (tokenInfo *const token, const tokenInfo *const class,
 
 		if (isKeyword (token, KEYWORD_async))
 			readToken (token);
-		else if (isType(token, TOKEN_KEYWORD) && isKeyword (token, KEYWORD_get))
+		else if (isType (token, TOKEN_KEYWORD) &&
+				 (isKeyword (token, KEYWORD_get) || isKeyword (token, KEYWORD_set)))
 		{
-			is_getter = true;
-			readToken (token);
-		}
-		else if (isType(token, TOKEN_KEYWORD) && isKeyword (token, KEYWORD_set))
-		{
-			is_setter = true;
-			readToken (token);
+			if (NextToken == NULL)
+			{
+				tokenInfo *savedToken = newToken ();
+				copyToken (savedToken, token, true);
+				readToken (token);
+				if (isType(token, TOKEN_OPEN_PAREN) && NextToken == NULL)
```

Comment:
I must verify when whether NextToken can be != NULL in readToken.
If there is such a control flow, we need a test case covering the case.
I must verify this.

Reply:
You are correct.  `NextToken` will always be `NULL`.  I updated the commit.

Reply:
Could you put Assert (NextToken == NULL);  ?